internal/api/core/describe.go

```diff
@@ -19,3 +29,26 @@ func SetAPIDescription(description APIDescription) {
 func GetAPIDescription() APIDescription {
 	return apiDescription
 }
+
+func GetDescriptionFromHandler(basePath string) (string, error) {
+	absPath := getHandlerFilePath(basePath)
+	if !util.IsFile(absPath) {
+		return "", fmt.Errorf("Unable to find file path to API Handler. Endpoint: '%s'. Expected path: '%s'.", basePath, absPath)
+	}
+
+	re := regexp.MustCompile(`Handle`)
+	description, err := util.GetDescriptionFromFunction(absPath, re)
```

Comment:
You can just pass the compile expression directly as the argument.

If you don't want to do that, then rename the variable to something more descriptive like "handlePattern".